Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsUpdateForBusinessQualityUpdateProfileWindows10/MSFT_IntuneWindowsUpdateForBusinessQualityUpdateProfileWindows10.psm1

```diff
@@ -94,9 +94,9 @@ function Get-TargetResource
 
             if (-not [System.String]::IsNullOrEmpty($DisplayName))
             {
-                $getValue = Get-MgBetaDeviceManagementWindowsQualityUpdateProfile `
-                    -Filter "DisplayName eq '$DisplayName'" `
-                    -ErrorAction SilentlyContinue
+                $getValue = Get-MgBetaDeviceManagementWindowsQualityUpdateProfile | Where-Object -FilterScript {
```

Comment:
You want to add ```-All``` here to retrieve all profiles first and maybe also keep ```-ErrorAction SilentlyContinue``` otherwise if the cmdlet fails you go straight to the next catch block instead of returning ```$nullResult```.

Reply:
Thanks for being the good eyed watcher. Just updated it, totally makes sense.